HW2/fastq_filtrator.py

```diff
@@ -0,0 +1,114 @@
+def read_from_file(input_fastq):
+    """
+    :param input_fastq: fastq file with reads
+    :return: list length = number of reads, on each position - sublist with information on
+    [0] name [1] sequence [2] separator [3] quality
+    """
+    reads_list = []
+    with open(input_fastq, 'r') as file:
+        lines = file.readlines()
+
+    for read_no in range((len(lines) // 4)):
+        if read_no == (len(lines) // 4) - 1 and lines[read_no * 4 + 3][-1] != "\n":
+            reads_list.append([lines[read_no * 4][:-1], lines[read_no * 4 + 1][:-1], lines[read_no * 4 + 2][:-1],
+                               lines[read_no * 4 + 3]])
+        else:
+            reads_list.append([lines[read_no*4][:-1], lines[read_no*4 + 1][:-1], lines[read_no*4 + 2][:-1],
+                               lines[read_no*4 + 3][:-1]])
+
+    return reads_list
+
+
+def GC_check(read, gc_bounds):
```

Comment:
В питоне любые переменные (кроме классов и глобальных констант) принято называть только маленькими буквами, даже если это аббревиатуры и т.д.
```suggestion
def gc_check(read, gc_bounds):
```